src/Ses/Input/Body.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace AsyncAws\Ses\Input;
+
+class Body
+{
+    /**
+     * An object that represents the version of the message that is displayed in email clients that don't support HTML, or
+     * clients where the recipient has disabled HTML rendering.
+     *
+     * @var Content|null
+     */
+    private $Text;
+
+    /**
+     * An object that represents the version of the message that is displayed in email clients that support HTML. HTML
+     * messages can include formatted text, hyperlinks, images, and more.
+     *
+     * @var Content|null
+     */
+    private $Html;
+
+    /**
+     * @param array{
+     *   Text?: \AsyncAws\Ses\Input\Text|array,
```

Comment:
The docblock on property says it's `Content`

Reply:
Yeah. But the class was not generated.

Reply:
maybe the class does not exist and the issue is the doc lock here ?

Reply:
Yeah, the doc block is wrong.